### ISSUE TITLE
Add command to print rai version

### ISSUE THREAD

**Issue by bradlo** [RelationalAI/rai-cli#3]
Add command `rai version`

**Comment by TeddyBear06** [comment]
Hi,

I would be pleased to contribute on this one.

I saw you're using Cobra as CLI framework (lovely one by the way).

How would you want to set rai's CLI versions ?

Two options comes in my mind:

- Hard-coded in a `version_current.go` file ([Hugo did this](https://github.com/gohugoio/hugo/blob/master/common/hugo/version_current.go) and it's easier to distribute for both devs and users)
- At compile time using `-ldflags="-X 'main.version=0.1.2-alpha'"` (may be better if you think about CI automations but require to update both `make` file and `README.md`)

Cobra support ["out of the box" version command flags](https://pkg.go.dev/github.com/spf13/cobra#Command.Version) by injecting a `Version`:

`var root = &cobra.Command{Use: "rai", Version: version}`

Then we can get rai's version using both `-v` or `--version` flags:

```
$ ./make
$ ./build/rai --version
rai version 0.1.1-alpha
$ ./build/rai -v
rai version 0.1.1-alpha
```

I'll send a pull request with a working POC in order to share my POV about this question.

**Comment by andy-dean-rai** [comment]
This feature would have been helpful for me today.

**Comment by andy-dean-rai** [comment]
I agree with needing a --version flag which users will often expect, but it would also be good to have a version command that can show the version of the associated database and engine. That information should also make it clear whether or not the CLI, database, and engine are all compatible.

**Comment by billscheidel-rai** [comment]
**Note:** This issue has been migrated to [https://relationalai.atlassian.net/browse/RAI-4917](https://relationalai.atlassian.net/browse/RAI-4917).

_This link is only accessible to employees of RelationalAI._
